test/JMeter/README.md

```diff
@@ -0,0 +1,15 @@
+**Stress Test Guide:**
+
+Stress testing on the ingress was performed using [JMeter](https://jmeter.apache.org/download_jmeter.cgi).
+Initial testing was performed in GUI mode, however if pushing the limits it can be run from CLI as well.
+
+In order to stress test open the `PerformanceTest.jmx` file with JMeter and edit these properties:
+- In HTTP Request Defaults, edit `Server Name or IP` field to point to your helloworld application.
+    Recall that you can find this by using `kubectl get kservice helloworld-sleep
```

Comment:
I think it's more involved then this one `kubectl` command. You likely need to output in JSON and parse that to extract the IP. Search for "Knative Service IP address" and you will find various help on the topic... including kubectl examples.